### ISSUE TITLE
Skipper & Cherenkov still looking for 'restock-engine-25-3-shrouds-b' texture

### ISSUE THREAD

**Issue by Poodmund** [PorktoberRevolution/ReStocked#754]
Receiving this message in the log:

```
[LOG 20:31:59.121] Load(Model): ReStock/Assets/Engine/restock-engine-skipper-1
[ERR 20:31:59.136] Texture 'ReStock/Assets/Engine/restock-engine-25-3-shrouds-b' not found!

[LOG 20:32:00.458] Load(Model): ReStockPlus/Assets/Engine/restock-engine-cherenkov-1
[ERR 20:32:00.474] Texture 'ReStockPlus/Assets/Engine/restock-engine-25-3-shrouds-b' not found!
```

This is one of the textures that was removed for v1.0.0 but it seems the .mu for each are still looking for it in a material(?)

Minor issue.

**Comment by ChrisAdderley** [comment]
63fb192045892cc64d42d629e5fef1fd61c31cf0 has a fix

**Comment by Poodmund** [comment]
What is really annoying is the KSP.log only reports the first missing instance of a material texture in a part so now that the 'shrouds-b' issue is resolved the log now reports:

```
[LOG 14:36:02.700] Load(Model): ReStock/Assets/Engine/restock-engine-skipper-1
[ERR 14:36:02.718] Texture 'ReStock/Assets/Engine/restock-engine-25-3-shrouds-n' not found!

[LOG 14:36:04.121] Load(Model): ReStockPlus/Assets/Engine/restock-engine-cherenkov-1
[ERR 14:36:04.136] Texture 'ReStockPlus/Assets/Engine/restock-engine-25-3-shrouds-n' not found!
```

**Comment by ChrisAdderley** [comment]
Should be fixed with latest commit, but needs dds conversion which will come later.

**Comment by ChrisAdderley** [comment]
Compressed so all good.

**Comment by Poodmund** [comment]
Did this all get resolved in the v1.0.1 release?
